docker-assets/initialize-httpd-auth.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+HTTPD_AUTH_CONFIG_DIR=/etc/httpd/auth-conf.d
+AUTH_CONFIG_FILE="${HTTPD_AUTH_CONFIG_DIR}/auth-configuration.conf"
+SSSD_CONF_FILE="/etc/sssd/sssd.conf"
```

Comment:
systemd service for this is fine, but what about other services that require an environment variable for startup (see upcoming auth-api service PR). https://github.com/abellotti/container-httpd/blob/d1f1141cbc384f22536c26e7e1b511886baffb98/docker-assets/entrypoint need a way to save the env before starting systemd.